chia/cmds/plotnft_funcs.py

```diff
@@ -270,21 +275,45 @@ async def submit_tx_with_confirmation(
         print(f"Error performing operation on Plot NFT -f {fingerprint} wallet id: {wallet_id}: {e}")
 
 
+async def wallet_id_lookup_and_check(wallet_client: WalletRpcClient, wallet_id: Optional[int]) -> int:
+    selected_wallet_id: int
```

Comment:
Don't know if this is stylistic, but you can hint the first one you assign to as well.  Feel free to resolve if you don't care.